src/Collections/DeltaConfigCollection.php

```diff
@@ -0,0 +1,276 @@
+<?php
+
+namespace SilverStripe\Config\Collections;
+
+use SilverStripe\Config\Middleware\DeltaMiddleware;
+
+/**
+ * Applies config modifications as a set of deltas on top of the
+ * middleware, instead of as modifications to the underlying list.
+ */
+class DeltaConfigCollection extends MemoryConfigCollection
+{
+    /**
+     * Key to disable config delta middleware
+     */
+    const NO_DELTAS = 8;
```

Comment:
Yes I picked this number since it was least likely to conflict; Unfortunately since this collection and it's middleware are tightly bound to one another, I had to put them into the same module.

Reply:
I've fixed this by having framework inject this value instead in CoreConfigFactory.